### PR TITLE
Add Pyschonauts API to Games & Comics

### PR DESCRIPTION
**Pull request by thamudi**
- a api contribution by thamudi for the game pyschonauts

- [ check] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [check ] My addition is ordered alphabetically
- [ check] My submission has a useful description
- [ Check] The description does not end with punctuation
- [Check ] Each table column is padded with one space on either side
- [ Check] I have searched the repository for any relevant issues or pull requests
- [ Check] Any category I am creating has the minimum requirement of 3 items
- [ check] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -488,6 +488,7 @@ API | Description | Auth | HTTPS | CORS |
 | [PlayerUnknown's Battlegrounds](https://pubgtracker.com/site-api) | PUBG Stats | `apiKey` | Yes | Unknown |
 | [Pokéapi](https://pokeapi.co) | Pokémon Information | No | Yes | Unknown |
 | [Pokémon TCG](https://pokemontcg.io) | Pokémon TCG Information | No | Yes | Unknown |
+| [Pyschonauts API](https://pyschonauts-api.netlify.app/) | Pyschonauts World Characters Information and PSI Powers | No | No | Yes |
```
Comment (pawelbr) [inline README.md:491]:
```suggestion
| [Pyschonauts](https://pyschonauts-api.netlify.app/) | Pyschonauts World Characters Information and PSI Powers | No | Yes | Yes |
```

Comment (pawelbr) [reply README.md:491]: No need to add API in the name. Ref. [contributing guide](https://github.com/public-apis/public-apis/blob/master/CONTRIBUTING.md)

**Review by pawelbr — CHANGES_REQUESTED** [review]
Please commit the suggestions stated below and don't forget to squash all the commits into a single one

File: README.md (hunk 1 above)
Comment (thamudi) [reply README.md:491]: Noted, thanks for the feedback!

Comment (thamudi) [reply README.md]: Alright, done! I squashed my commits. But some new ones also popped up that weren't mine, if you check there's a commit that says: "Add URLHaus API to Anti-Maleware"., by FH089.

Its my first time squashing.

**Comment by pawelborkar** [comment]
@thamudi 
I think there's something that went wrong while squashing the commits. As the commits from FH089 are 15days old and are already into the master branch. If not then there might be a bug in GitHub itself as they are testing some new features on the platform. 
Whatever the case is I will encourage you to close this PR and make a new one with the following suggestions.

**Pull request closed without merging**